src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -41,9 +43,64 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
+const fetchAndSetBoMo = async (req: Request, appData: ApplicationData) => {
+  if (!hasFetchedBoAndMoData(appData)) {
+    if (!appData.update) {
+      appData.update = {};
+    }
+    appData.update.review_beneficial_owners_individual = [];
+    appData.update.review_beneficial_owners_corporate = [];
+    appData.update.review_beneficial_owners_government_or_public_authority = [];
+
+    await retrieveBeneficialOwners(req, appData);
+    await retrieveManagingOfficers(req, appData);
+    setFetchedBoMoData(appData);
+  }
+};
+
+const checkBOIndividualValidation = (boi: BeneficialOwnerIndividual) => {
+  if (boi.usual_residential_address){
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
Do we need to take the is residential address same as correspondance/service address boolean into account?

return (is_residential_address_same_as_correspondace_address  || boi.usual_residential_address) ? true : false;

Should also check boi.date_of_birth has a day as well as month/year.